### PR TITLE
Add exception handling for WebSocketConnectionClosedException

### PR DESCRIPTION
**Pull request by Kileak**
Just catch the WebSocketConnectionClosedException and reinitialize the bot.

### CODE REVIEW

File: server/botserver.py [hunk 1]
```diff
@@ -94,32 +96,37 @@ def load_bot_data(self):
...
+                        command, channel, user = self.parse_slack_message(message)
+
+                        if command:
+                            log.debug("Received bot command : {} ({})".format(command, channel))
+                            HandlerFactory.process(self.slack_wrapper, self,
+                                                   command, channel, user)
+
+                        time.sleep(READ_WEBSOCKET_DELAY)
+                else:
+                    log.error("Connection failed. Invalid slack token or bot id?")
```
Comment (Grazfather) [inline server/botserver.py]: If this happens we don't want to be stuck in a loop: We should break.

Comment (Grazfather) [reply server/botserver.py]: (Please test with a bogus key?)

**Comment by Kileak** [comment]
Good catch. Though this change also shows problems with the way, the console thread is handled.

Even though the bot will not go into an endless loop anymore now, you'll have to press Enter once to get out of the input, which lets the console thread hang.

Maybe the console thread should also use a non-blocking way of reading from console, but I'd rather not implement this yet (also even more, since this is a special case).

**Comment by Grazfather** [comment]
Yeah we can live with it for now.

**Pull request merged**